### PR TITLE
makin uptime.js

### PR DESCRIPTION
**Pull request by GameModerator**

### CODE REVIEW

File: src/plugins/commander/commands/uptime.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            '!runtime',
+            '!uptime',
+        ];
+    }
+
+    call(message) {
+        const client = bot.client;
+        const time = client.uptime;
+
+        const toTimeString = (time) => {
```
Comment (Dorumin) [inline src/plugins/commander/commands/uptime.js:20]: Why not move this to be a method in the command class?

File: src/plugins/commander/commands/uptime.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        this.desc = `..Tells you how long the bot has been up`;
+        this.usages = [
+            '!runtime',
+            '!uptime',
+        ];
+    }
+
+    call(message) {
+        const client = bot.client;
+        const time = client.uptime;
```
Comment (Dorumin) [inline src/plugins/commander/commands/uptime.js:18]: These are only used once each, don't alias them

File: src/plugins/commander/commands/uptime.js [hunk 3]
```diff
@@ -13,33 +13,30 @@ class RuntimeCommand extends Command {
...
-            return `${hours}, ${minutes} and ${seconds}`;
+            return `${days}, ${hours}, ${minutes} and ${seconds} `;
         }
 
-        const uptime = toTimeString(time);
+        return `${hours}, ${minutes} and ${seconds}`;
+    }
+
+    call(message) {
+        const uptime = this.toTimeString(bot.client.uptime);
```
Comment (Dorumin) [inline src/plugins/commander/commands/uptime.js:39]: `bot` is a field on the command, this will throw at runtime

Comment (GameModerator) [reply src/plugins/commander/commands/uptime.js]: done

**Comment by Dorumin** [comment]
Ultra radical

**Pull request merged**